test/plots/cars-mpg.js

```diff
@@ -0,0 +1,35 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+
+export default async function() {
+  const data = await d3.csv("data/cars.csv", d3.autoType);
+
+  return Plot.plot({
+    x: {inset: 20, ticks: 10},
+    y: {grid: true},
+    color: {
+      type: "categorical",
+      legend: true // as soon as it's available
```

Comment:
```suggestion
      type: "categorical"
```

I 100% agree this is wanting a color legend, but I’d prefer not to add options until they’re implemented. We don’t want tests to break when legends are implemented.